process.yml

```diff
@@ -0,0 +1,112 @@
+# Orb 'circleci/python@2.1.1' resolved to 'circleci/python@2.1.1'
```

Comment:
What is this file doing?  Is this still needed?  (I see it has references to the old `foundational_work` directories...)